sumologic/resource_sumologic_muting_schedules_library_muting_schedule.go

```diff
@@ -301,6 +310,33 @@ func getScheduleDefinition(d *schema.ResourceData) ScheduleDefinition {
 	return scheduleDefinition
 }
 
+func StringIsValidStartDate() schema.SchemaValidateFunc {
```

Comment:
if im not wrong the validation of regex ensures if it is following the date format. this is doing -24 hr validation, do u think renaming the method would represent this correctly

Reply:
updated.